src/hotspot/share/cds/heapShared.cpp

```diff
@@ -116,6 +116,14 @@ GrowableArrayCHeap<oop, mtClassShared>* HeapShared::_pending_roots = NULL;
 narrowOop HeapShared::_roots_narrow;
 OopHandle HeapShared::_roots;
 
+#ifdef ASSERT
+bool HeapShared::is_archived_object_during_dumptime(oop p) {
+  assert(HeapShared::is_heap_object_archiving_allowed(), "must be");
+  assert(DumpSharedSpaces, "must be called during dump time only");
```

Comment:
I'd suggest to change it to the following to be consistent with asserts in other places:
`assert(DumpSharedSpaces, "this function is only used with -Xshare:dump");`

Reply:
Fixed.